cmd/operator/main.go

```diff
@@ -148,9 +150,13 @@ func main() {
 	//       with cluster scoped resources. Once those issues are resolved, it may be worth switching to using that
 	//       cache type.
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metrics.Host, metrics.Port),
-		Port:               9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: fmt.Sprintf("%s:%d", metrics.Host, metrics.Port),
+		},
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
```

Comment:
Not sure if you saw, this is the default value https://github.com/kubernetes-sigs/controller-runtime/blob/c93e2abcb28eb71fccad7dc565f0547cc07e5566/pkg/webhook/server.go#L78

Reply:
I missed that. I will remove this line then.